hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveSchemaUtil.java

```diff
@@ -147,13 +147,12 @@ public void testSchemaConvertToIcebergSchemaForEveryPrimitiveType() {
   @Test
   public void testNotSupportedTypes() {
     for (FieldSchema notSupportedField : getNotSupportedFieldSchemas()) {
-      AssertHelpers.assertThrows(
-          "should throw exception",
-          IllegalArgumentException.class,
-          "Unsupported Hive type",
-          () -> {
-            HiveSchemaUtil.convert(Lists.newArrayList(Arrays.asList(notSupportedField)));
-          });
+      Assertions.assertThatThrownBy(
+              () -> {
```

Comment:
nit: can we remove the {} wrapping here?

Reply:
make sense, thanks!